src/App.tsx

```diff
@@ -30,6 +40,8 @@ export function App() {
     ? resolveLanguage(company.settings.language_id)
     : undefined;
 
+  const isLoginPage = location.pathname.startsWith('/login');
```

Comment:
Can we inline this if we don't reuse this variable?

Reply:
sure, made inline